demos/functions/views/demo/workbox-background-sync/sw.hbs

```diff
@@ -0,0 +1,14 @@
+{{{WORKBOX_SW_IMPORT}}}
+
+const queue = new workbox.backgroundSync.Queue('myQueueName');
+
+self.addEventListener('fetch', (event) => {
+  const pathname = new URL(event.request.url).pathname;
+  if (pathname === '/demo/workbox-background-sync/example.txt') {
```

Comment:
I feel we should show how to add this to queue when fetch fails.

demo might make more sense that ways

Reply:
Do you fancy taking on this issue and doing this? I agree it makes more sense (although I must confess the current flow actually works as it is, so why wait until an error when you can just queue it?)

Reply:
Sure thing, i'll get it done. But the reason i'd like it this way is because

a.) Its more close to the way it would be used at times

b.) It shows how it would actually play after network coming back